README.md

```diff
@@ -50,4 +50,26 @@ Currently there's no vendoring provided for this project.
 * **vulcan-unclassified** - Example vulnerability to test the monitoring of unclassified vulnerabilities - not for production
 * **vulcan-vulners** - Runs https://vulners.com/api/v3/burp/software/
 * **vulcan-wpscan** - Checks Wordpress sites for vulnerabilities using the open source wpscan utility
-* **vulcan-zap** - Checks for vulnerabilities in web applications using OWASP ZAP
\ No newline at end of file
+* **vulcan-zap** - Checks for vulnerabilities in web applications using OWASP ZAP
+
+## Building and testing
+
+This project is primarily built using the [vulcan-checks-bsys](https://github.com/adevinta/vulcan-checks-bsys) project.
+
+But it's possible to build each one of the checks with `go build`.
+
+In every check directory there is an example configuration file called `local.toml.example`, most checks reads a
+file named `local.toml` if you pass the `-t` parameter, so by copying that file you can do a local test of your code
+before you commit.
+
+The checks that are not self-contained, and instead relay on external binaries, can be locally tested with
+`vulcan-checks-bsys`. By running `vulcan-build-images -r PATH_TO_THE_CHECK_DIR` it will create a docker image and
+run it, feeding the check the input from local.toml.
```

Comment:
Suggestion: `s/check the input/check with the input/`

Reply:
fixed